src/js/components/TextArea/TextArea.js

```diff
@@ -17,10 +17,6 @@ const TextArea = forwardRef(
         ? valueProp
         : (formContext && name && formContext.get(name)) || '',
     );
-    useEffect(() => setValue(valueProp), [valueProp]);
-    useEffect(() => {
-      if (formContext && name) setValue(formContext.get(name) || '');
```

Comment:
We still need something like this to initialize the value from the Form `value` object.